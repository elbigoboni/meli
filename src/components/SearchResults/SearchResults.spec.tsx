import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router-dom";
import { SearchResults } from ".";

describe("<SearchResults />", () => {
  describe("Render", () => {
    test("should render correctly", () => {
      const { container } = render(<SearchResults />, {
        wrapper: MemoryRouter,
      });
      expect(container).toMatchSnapshot();
    });
  });
});
