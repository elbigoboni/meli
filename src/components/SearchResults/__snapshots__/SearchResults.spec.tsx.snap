// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResults /> Render should render correctly 1`] = `
<div>
  <div
    class="sc-fubCfw iaiWza"
  />
</div>
`;
